### PR TITLE
add tmux command documentation

### PR DESCRIPTION
**Pull request by laurenshen**
# Advanced HW 5 submission

Name: Lauren Shen
Uniqname: SHENLA

### CODE REVIEW

**Comment by laurenshen** [comment]
I fixed it, and compared it to the cd documentation to make sure it was similar!

File: _commands/tools/tmux.md [hunk 1]
```diff
@@ -1,47 +1,44 @@
...
+<!--more-->
+
 ### Useful Options / Examples
 
-#### CTRL-B + % or CTRL-B + "
 
-##### % represents a vertical split, while " represents a horizontal split. This will split the terminal window into multiple terminal sessions. Use CTRL-B + arrow keys to navigate.
+#### `CTRL-B + % or CTRL-B + "`
+
+% represents a vertical split, while " represents a horizontal split. This will split the terminal window into multiple terminal sessions. Use CTRL-B + arrow keys to navigate.
```
Comment (tarunsk) [inline _commands/tools/tmux.md]: Could we make `%` and `CTRL-B + arrow keys` inline code sections using back-ticks (like I've done in this review)?

File: _commands/tools/tmux.md [hunk 2]
```diff
@@ -1,47 +1,44 @@
...
 
-#### tmux new -s foo
+#### `tmux new -s foo`
 
-##### New creates a session with the name foo and attaches it to the terminal. Adding -d to the end will create a detached session named foo. To list sessions, use the command tmux ls.
+New creates a session with the name foo and attaches it to the terminal. Adding -d to the end will create a detached session named foo. To list sessions, use the command tmux ls.
```
Comment (tarunsk) [inline _commands/tools/tmux.md]: Same with `-d` and `tmux ls`

File: _commands/tools/tmux.md [hunk 3]
```diff
@@ -1,47 +1,44 @@
...
+#### `CTRL-B + d`
 
-##### Detaches from session.
+Detaches from session.
 
-#### tmux kill-session -t foo
+#### `tmux kill-session -t foo`
 
-##### Kills session named foo. To kill all sessions except for the current one, use tmux kill-session -a. To kill all sessions except for the session named foo, use tmux kill-session -a -t foo.
+Kills session named foo. To kill all sessions except for the current one, use tmux kill-session -a. To kill all sessions except for the session named foo, use tmux kill-session -a -t foo.
```
Comment (tarunsk) [inline _commands/tools/tmux.md]: Same with `tmux kill-session -a` and `tmux kill-session -a -t foo`

**Comment by cyanliu** [comment]
You've been awarded credit for advanced homework 5 and given 2 points of extra credit for creating your initial PR more than 2 days before the deadline 🎉 

You won't see this immediately reflected on Canvas, but it will be there the next time we update grades!

**Pull request merged**